tt_metal/impl/dispatch/kernels/cq_dispatch.cpp

```diff
@@ -897,7 +897,7 @@ static inline bool process_cmd_d(uint32_t &cmd_ptr) {
             break;
 
         case CQ_DISPATCH_CMD_WRITE_PACKED: {
-            DPRINT << "cmd_write_packed" << ENDL();
+            // DPRINT << "cmd_write_packed" << ENDL();
```

Comment:
Are we intentionally removing DPRINT statements from dispatcher?

Reply:
oh I was running into code size issues when compiling this and running dprint tests on idle eth, I can try adjusting the sizes for BH and enable the comments